js/data/index.d.ts

```diff
@@ -11,18 +11,29 @@ export type SearchOperation = '=' | '<>' | '>' | '>=' | '<' | '<=' | 'startswith
 
 type KeySelector<T> = string | ((source: T) => string | number | Date | Object);
 
-type BaseGroupDescriptor<T> = {
+type SelectionDescriptor<T> = {
     selector: KeySelector<T>;
 };
 
+type OrderingDescriptor<T> = SelectionDescriptor<T> & {
+    desc?: boolean;
+};
+
+/**
+ * @docid
+ * @public
+ */
```

Comment:
Discovering unions has problems with `@docid` tag now, let's skip it for now (we probably will remove it from the `SearchOperation` as well
```suggestion
/**
 * @public
 */
```

Reply:
fixed